aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -246,5 +247,41 @@ Tensor& logical_xor_(Tensor& self, const Tensor& other) {
   return native::logical_xor_out(self, self, other);
 }
 
+Tensor& lt_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  TORCH_CHECK(result.scalar_type() == kBool || result.scalar_type() == kByte,
+      "The output tensor of lt must be a bool or byte tensor.");
+  if (result.dtype() == at::ScalarType::Byte) {
+    AT_WARN("torch.lt received 'out' parameter with dtype torch.uint8, this behavior is now deprecated," \
```

Comment:
@izdeby can we remove the deprecation warning now, or was the plan to remove in an even more future release?

Reply:
We can remove it now. I will have a PR which will kill all of these sometime soon so go for it, just keep in mind that it will fail some tests.

Reply:
Fixed